packages/react-provider/src/components/FluentProvider/useFluentProvider.ts

```diff
@@ -24,9 +24,21 @@ export const useFluentProvider = (props: FluentProviderProps, ref: React.Ref<HTM
    * nesting providers with the same "dir" should not add additional attributes to DOM
    * see https://github.com/microsoft/fluentui/blob/0dc74a19f3aa5a058224c20505016fbdb84db172/packages/fluentui/react-northstar/src/utils/mergeProviderContexts.ts#L89-L93
    */
-  const { dir = parentContext.dir, targetDocument = parentContext.targetDocument, theme = {} } = props;
+  const { dir = parentContext.dir, targetDocument = parentContext.targetDocument, theme } = props;
   const mergedTheme = mergeThemes(parentTheme, theme);
 
+  React.useEffect(() => {
+    if (process.env.NODE_ENV !== 'production' && mergedTheme === undefined) {
+      // eslint-disable-next-line no-console
+      console.warn(`
+      FluentProvider: your theme is not set !
```

Comment:
```suggestion
      FluentProvider: you don't defined "theme".
```

Can we make it more friendly? 😀

Reply:
hahah we can, I'm becoming a machine I guess. SET SET GET BOBBA FETT :D